NebulaWorld/SimulatedWorld.cs

```diff
@@ -205,8 +205,6 @@ public void UpdateRemotePlayerAnimation(PlayerAnimationUpdate packet)
                 if (remotePlayersModels.TryGetValue(packet.PlayerId, out RemotePlayerModel player))
                 {
                     player.Animator.UpdateState(packet);
-
-                    player.Effects.UpdateState(packet);
```

Comment:
Isn't this needed to play the VFX like mining torch and thrusters?

Reply:
I move the function call to RemotePlayerAnimation.UpdateState() so they can share the same packet buffer.